### DISCUSSION TITLE
:pray: Support url shorteners (ex bilty) while select from httfs :pinching_hand:

### DISCUSSION THREAD

**Discussion by adriens** [duckdb/duckdb#6653] [Ideas]
Hi, I would like to be able to use url shorteners to make sql easier to copy.

See current working sql : 

```sql
INSTALL httpfs;
LOAD httpfs;
select * from 'https://raw.githubusercontent.com/adriens/odata-eaux-baignade-noumea/main/data/historic.csv';
```

and i would like to transform it to : 

```sql
INSTALL httpfs;
LOAD httpfs;
select * from 'https://bit.ly/3mAUIPr';
```

... but the url shorteners is not suported : 

<pre>D <font color="#8AE234"><b>INSTALL </b></font>httpfs;
D <font color="#8AE234"><b>LOAD </b></font>httpfs;
D <font color="#8AE234"><b>select </b></font>* <font color="#8AE234"><b>from </b></font><font color="#C4A000">&apos;https://bit.ly/3mAUIPr&apos;</font>;
Error: Catalog Error: Table with name https://bit.ly/3mAUIPr does not exist!
Did you mean &quot;pg_type&quot;?
LINE 1: select * from &apos;https://bit.ly/3mAUIPr&apos;;
                      ^
</pre>

**Comment by Mause** [comment]
The reason you're getting an error there is because the URL no longer ends with .csv - try explicitly calling it with `read_csv_auto`?

**Reply by adriens** [reply]
![image](https://user-images.githubusercontent.com/5235127/224213553-0f6a34af-ee3d-4dc9-9d0f-7885bdeb80df.png)

**Reply by adriens** [reply]
Nice shot :clap: Thanks a lot :pray:
